transaction/core/src/account_keys.rs

```diff
@@ -13,37 +13,59 @@
 #![allow(non_snake_case)]
 
 use crate::{domain_separators::SUBADDRESS_DOMAIN_TAG, view_key::ViewKey};
-use alloc::string::{String, ToString};
-use blake2::{Blake2b, Digest};
+
 use core::{
     cmp::Ordering,
     fmt,
     hash::{Hash, Hasher},
 };
-use curve25519_dalek::scalar::Scalar;
+
+use alloc::{
+    string::{String, ToString},
+    vec::Vec,
+};
+
 use mc_crypto_digestible::Digestible;
 use mc_crypto_keys::{RistrettoPrivate, RistrettoPublic};
 use mc_util_from_random::FromRandom;
+
+use blake2::{Blake2b, Digest};
+use curve25519_dalek::scalar::Scalar;
 use prost::Message;
 use rand_core::{CryptoRng, RngCore};
-use serde::{Deserialize, Serialize};
 
 /// An account's "default address" is its zero^th subaddress.
 pub const DEFAULT_SUBADDRESS_INDEX: u64 = 0;
 
+/// An account's version number. If breaking changes are made to the interpretation
+/// of account keys or public addresses, this should be incremented so that old clients
+/// will fail fast on data they shouldn't handle.
+pub const CURRENT_ACCOUNT_KEY_VERSION: u32 = 0;
```

Comment:
This should probably start from `1` since `0` is a default grpc value and could indicate a misbehaving implementation.